data-reconciliation-app/README.md

```diff
@@ -1,22 +1,29 @@
 # CCF Data Reconciliation Application
 
-This is the repo for _CCF Data Reconciliation - sample_.
+This is the _CCF Data Reconciliation - sample_ in typescript.
 
 ## Overview
 
 The CCF network will be used to host a reconciliation service where different parties with membership (banks and data providers) will be able to submit their own data to be reconciled against "each other's data" in a confidential manner without exposing the data to other members in the network.
 
-The sample will use the voting process to reconcile members' data; on the data submission, when new record is submitted the app will check if it does not exist in the key-value store, it will be added; otherwise, a vote is added to this record with a member ID, and the vote will be "agree" if data attributes match; otherwise, it will "disagree."
+When a new record(s) is submitted through ingestion endpoints, the application will search the key-value store by the record's key; if this key does not exist, it will be added; otherwise, a vote is added to this record with a member ID and the submitted value.
 
 ## Architecture
 
-This is a work in progress overview of the current architecture and key interactions and integrations.
+The reconciliation application consists of three main parts.
```

Comment:
As we have a new title right after the sentence, it seems dangling... 
Suggestion (can be ignored if you disagree):

```suggestion
The reconciliation application consists of three main parts: Data Ingestion, Data Reconciliation and Data Reporting.
```